src/lib/pages/backoffice/Actions/CheckOut/CheckOutResults/CheckOutResults.js

```diff
@@ -0,0 +1,49 @@
+import { Loader } from '@components/Loader';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Header, Segment } from 'semantic-ui-react';
+import { ItemList } from './ItemList';
+import { PatronList } from './PatronList';
+
+export default class CheckOutResults extends Component {
+  componentDidMount() {
+    const { clearResults } = this.props;
+    clearResults();
+  }
+
+  render() {
+    const { itemList, isLoading, patronList } = this.props;
+
+    // NOTE: if we have 1 result we will get redirected, avoid render list with 1
```

Comment:
can you just mention in this comment that we will be redirected straight to the details page of the patron/item?